src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Drawable {
+    default void draw() {
```

Comment:
let's make this method abstract, so it will be necessary to override it in implementations